### PR TITLE
QA Test plan

### PR DESCRIPTION
**Pull request by rbillings**

### CODE REVIEW

**Comment by groovecoder** [comment]
Thanks @rbillings! I'll check this out tomorrow.

**Comment by rbillings** [comment]
Suggest updating copy to "modal window" instead of pop-up, match all text with Copy document

**Comment by pdehaan** [comment]
FWIW, in a few of the other TestPilot/related experiments, we threw markdown files in a /docs/ directory, which kept everything together.

File: tests/testplan.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+## ENVIRONMENTS:
+
+| ENVIRONMENT | URL |
+|:------------|:----|
+| Development | http://testpilot.dev.mozaws.net/
+
+
+## PREREQUISITES:
+
+- Use Firefox 47 or later
```
Comment (groovecoder) [inline tests/testplan.md]: Needs to be Firefox 48

File: tests/testplan.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+1. Open Firefox (any channel with Firefox 47 or higher, _Release_ or _Nightly_).
+
+1. Go to http://testpilot.dev.mozaws.net/
+
+1. Install Test Pilot. After installation, select "Blok" Add-on and enable it.
+
+1. Open a new tab and load http://cnn.com. 
+	1. Verify that Blok header displays with text confirming it is enabled. 
+	1. Verify Blok shows total blocked
```
Comment (groovecoder) [inline tests/testplan.md]: No longer shows total # of blocked trackers/requests. Simply shows "Blocked trackers"

**Comment by groovecoder** [comment]
Thanks @rbillings. Just a couple little notes.

**Comment by groovecoder** [comment]
@rbillings are you able to make these changes? If not I can take it on myself today while I wait for Telemetry folks to set me up with Telemetry end-to-end access.

**Comment by rbillings** [comment]
@groovecoder I will do it today- I was waiting to see if there was any more feedback from the email, but it seems this is it.

**Comment by rbillings** [comment]
I've moved it to a /docs folder [great suggestion!] and addressed all other comments.

**Comment by pdehaan** [comment]
Ruh roh, now it looks like we have a /docs/testplan.md and /tests/testplan.md (and a pesky .DS_Store).

**Comment by pdehaan** [comment]
LGTM.
Feel free to merge, unless @groovecoder has any other feedback.

**Comment by pdehaan** [comment]
Actually, I'll merge since I have access and am drunk with power. 🍻

**Comment by rbillings** [comment]
Thanks @pdehaan & @groovecoder !

**Pull request merged**